.github/workflows/continuous-delivery.yaml

```diff
@@ -39,6 +39,16 @@ jobs:
           commit-assets: |
             ./dist
           node-module: true
+          release-rules: |
+            [
+              { "release": "patch", "type": "build" },
+              { "release": "patch", "type": "chore", "scope": "deps" },
+              { "release": "patch", "type": "chore", "scope": "deps-dev", "message": "bump typescript from*" },
+              { "release": "patch", "type": "ci" },
+              { "release": "patch", "type": "docs" },
```

Comment:
Should docs trigger a release?

Reply:
Nope! Resolved!

I'm thinking now, should we also whitelist `@zeit/ncc` and `@ridedott/eslint-config`? They could both change the compiled output, albeit, only aesthetically and not functionally.

Reply:
`eslint` does not automatically change your code. It requires a human intervention to. As for ncc 🤷‍♂️

Reply:
Alright, I'm leaving them out. We will add them later on if we notice they are being missed.

Reply:
@mad-it @acazacu ncc _does_ change the output. This should trigger a release.